src/dataframe/dataframe.jl

```diff
@@ -1164,6 +1164,9 @@ function Base.append!(df1::DataFrame, df2::AbstractDataFrame; cols::Symbol=:sete
     nrows, ncols = size(df1)
     targetrows = nrows + nrow(df2)
     current_col = 0
+    # in the code below we use a direct access to _columns because
```

Comment:
There's the same pattern in `push!`, right?

Reply:
right. added